homeassistant/components/lcn/manifest.json

```diff
@@ -2,9 +2,9 @@
   "domain": "lcn",
   "name": "LCN",
   "codeowners": ["@alengwenus"],
-  "config_flow": false,
+  "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/lcn",
   "iot_class": "local_push",
   "loggers": ["pypck"],
-  "requirements": ["pypck==0.7.17"]
+  "requirements": ["pypck==0.7.19", "lcn-frontend==0.1.2"]
```

Comment:
Can the bump of pypck happen in a separate PR

Reply:
Yes, that's possible. The newest changes are not necessary for this PR.